{"status":"FAIL","msg":"FAIL:\n Uncaught: { message: \"Expected a RangeError but got a TypeError\" }\n    at ../../test262/harness/assert.js:95\n    at anonymous (../../test262/test/built-ins/Temporal/Duration/prototype/toString/roundingmode-invalid-string.js:12)\n\n","path":"../../test262/test/built-ins/Temporal/Duration/prototype/toString/roundingmode-invalid-string.js"}